main.tf

```diff
@@ -1,40 +1,47 @@
 module "networking" {
   source = "./modules/networking"
 
-  app-name                = local.app-name
-  id_of_vpc               = module.networking.id_of_vpc
-  both_private_subnets_id = [module.networking.both_private_subnets_id]
+  app_name             = local.app_name
+  availability_zones   = ["us-west-2a", "us-west-2b", "us-west-2c"]
+  cidr                 = "10.0.0.0/16"
+  public_subnets_list  = ["10.0.1.0/24", "10.0.2.0/24", "10.0.3.0/24"]
+  private_subnets_list = ["10.0.4.0/24", "10.0.5.0/24", "10.0.6.0/24"]
+  just_count           = 3
+  enable_db_subnets    = true
 }
 
 module "db" {
-  source                     = "modules/db"
-  app-name                   = local.app-name
-  id_of_vpc                  = module.networking.id_of_vpc
+  source                     = "./modules/db"
+  app_name                   = local.app_name
+  availability_zones         = ["us-west-2a", "us-west-2b", "us-west-2c"]
+  database_name              = "dbname"
+  vpc_id                     = module.networking.vpc_id
   vpc_rds_security_group_ids = [module.sg.vpc_rds_security_group_ids]
-  both_db_subnets_name       = module.networking.both_db_subnets_name
-  master_password            = local.master-password
+  db_subnets_name            = module.networking.db_subnets_name
+  master_username            = "stoyandg"
+  master_password            = local.master_password
```

Comment:
please create a new resource `aws_secretsmanager_secret` for storing the password, put the value in it manually ( using AWS UI ) and read the value using `data "aws_secretsmanager_secret"` 

more info about the service [here](https://docs.aws.amazon.com/secretsmanager/latest/userguide/intro.html)
terraform resource doc is [here](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/secretsmanager_secret)
terraform datasource doc is [here](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/secretsmanager_secret.html)

Reply:
Done.